app.vue

```diff
@@ -1,5 +1,5 @@
 <template>
   <div>
-    <NuxtWelcome />
+    <NuxtPage />
   </div>
```

Comment:
you should setup the layout instead, so that you have not add navbar and footer again and again.
refer
https://nuxt.com/docs/guide/directory-structure/layouts

Reply:
Have resolved this comment